app/models/activity.rb

```diff
@@ -82,6 +82,16 @@ class Activity < ApplicationRecord
                                    .confirmed.by_dates(period_begin, period_end)
                                    .order(occur_at: 'desc')
                                }
+  def accompaniable?
```

Comment:
I am a little bit worried about developers confusing this method and the activity method 'accompaniment_eligible?' since the method name meaning is similar.  Was thinking maybe this could be framed as 'accompaniment_limit_met?'